app/main.go

```diff
@@ -92,6 +92,9 @@ var (
 	boltdbPath = kingpin.Flag("boltdb-path", "path to BoltDB store, e.g. /data/bolt.db").Envar("BOLTDB_PATH").String()
 
 	ver = kingpin.Flag("version", "print the current version revision").Bool()
+
+	monitoringBackends = kingpin.Flag(
```

Comment:
If running standalone it seems that stackdriver collector can get initialized without required attributes (project) and error loop:
```
λ docker run -p 8080:8080 \                                                                                                                                                                                               (2)
 -v ${PWD}/metrics.yaml:/ts-bridge/metrics.yaml \
 -v ~/.gcp/graphite.json:/ts-bridge/graphite.json \
 -e "GOOGLE_APPLICATION_CREDENTIALS=/ts-bridge/graphite.json" \
 ts-bridge:e1df8f6 \
 --debug \
 --storage-engine=boltdb \
 --enable-status-page \
 --update-parallelism=4 \
 --sync-period=10s \
 --metric-exporters=stackdriver

...
time="2020-11-25T22:37:18Z" level=error msg="error running sync() routine: errors occured during sync init: [unable to initialize stats collector: error initializing stats collector - project empty: set SD_PROJECT_FOR_INTERNAL_METRICS or --stats-sd-project if not running on App Engine]"
```

Reply:
It actually causes a segfault down the line:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0xda1ead]

goroutine 12 [running]:
github.com/google/ts-bridge/tsbridge.(*Metric).Update(0xc0006d7e40, 0x11f0380, 0xc0005a5f20, 0x11ebf80, 0xc0006d0be0, 0x0, 0x0, 0x0)
	/go/src/ts-bridge/tsbridge/metric.go:115 +0x6d
github.com/google/ts-bridge/tsbridge.UpdateAllMetrics.func2(0xc00027e850, 0xc0000a7810, 0x11f0380, 0xc0005a5f20, 0x11ebf80, 0xc0006d0be0, 0x0, 0xc00045cba0, 0xc0006d7e40)
	/go/src/ts-bridge/tsbridge/metric.go:78 +0xc5
created by github.com/google/ts-bridge/tsbridge.UpdateAllMetrics
	/go/src/ts-bridge/tsbridge/metric.go:74 +0x292
```

Reply:
This might not have been introduced by this change though, so feel free to push back and fix separately.

Reply:
I'll open a separate issue for this.

Reply:
👍